node/src/components/linear_chain.rs

```diff
@@ -412,12 +454,12 @@ where
                 effects.extend(
                     effect_builder
                         .handle_linear_chain_block(*block.clone())
-                        .map_some(move |fs| Event::FinalitySignatureReceived(Box::new(fs))),
+                        .map_some(move |fs| Event::FinalitySignatureReceived(Box::new(fs), true)),
```

Comment:
Isn't this where it should be `false`? This is a signature we just created, isn't it, so it's _not_ external/gossiped?

Reply:
ups.

Reply:
Maybe make the tuple variant a struct variant? Idk what the bool means at a glance.